### ISSUE TITLE
Lack of a Skip Link

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#55]
👪 Related issue: [#106](https://github.com/frameless/developer.overheid.nl/issues/106)

**Pagina:**
[Rijkshuistijl Community Template](https://rijkshuisstijl-community-templates-git-102644-nl-design-system.vercel.app/developer-overheid/homepage)
[Zoek API's Template](https://rijkshuisstijl-community-templates-eviwhsy01-nl-design-system.vercel.app/developer-overheid/api-overzicht)

**Probleembeschrijving:**
De pagina mist een Skip Link bovenaan om snelle navigatie van de pagina voor toetsenbordgebruikers mogelijk te maken. De Skip Link moet normaal gesproken helemaal aan het begin van het header element worden geplaatst of er net voor, zodat het het eerste focusbare element op de pagina is.

**Gerelateerde WCAG:**
WCAG 2.4.1 Bypass Blocks

**Referentieafbeelding:**

<img width="521" alt="the tabbing order of the first focusable in a website's navigation are enumerated, the first one is not a Skip Link component" src="https://github.com/user-attachments/assets/23f938bf-bd60-4617-9057-3beaf44e37f7" />

**To-do's**

- [x] skip link toevoegen op homepage (consistent op iedere pagina)
- geplaatst in code vóór de page header. Linkt door naar eerste interactieve element in main 
- [ ] skip link toevoegen op API overzicht (consistent op iedere pagina)
